<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DebugChannelHigh</class>
 <widget class="QWidget" name="DebugChannelHigh">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>463</width>
    <height>215</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>20</number>
   </property>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <property name="verticalSpacing">
      <number>6</number>
     </property>
     <item row="0" column="3">
      <widget class="QLabel" name="label_4">
       <property name="styleSheet">
        <string notr="true">color:white</string>
       </property>
       <property name="text">
        <string>参考值</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="lab_noise_h1">
       <property name="text">
        <string>噪声偏置</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="lab_zero_h1">
       <property name="text">
        <string>中心偏置</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="TTKCheckButtonWidget" name="cbx_FIR_h1">
         <property name="text">
          <string>FIR</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="TTKCheckButtonWidget" name="cbx_wavelet_h1">
         <property name="text">
          <string>小波</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="lab_filter_h1">
       <property name="text">
        <string>前置滤波</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLineEdit" name="lineEdit_H1_ZERO">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>22</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit,QDateTimeEdit{
border:0px solid #242424;
border-radius:3px;
padding:2px;
color:#FFFFFF;
selection-background-color:#1C1C28;
selection-color:#0EC3FD;
}</string>
       </property>
       <property name="text">
        <string notr="true"/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QLabel" name="lab_offset_zero_h1">
       <property name="baseSize">
        <size>
         <width>20</width>
         <height>0</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">color:white</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::Panel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="text">
        <string notr="true">0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QLabel" name="lab_offset_noise_h1">
       <property name="baseSize">
        <size>
         <width>20</width>
         <height>0</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">color:white</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::Panel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="text">
        <string notr="true">0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_H1">
       <property name="text">
        <string>TEV1</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLineEdit" name="lineEdit_H1_NOISE">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>22</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit,QDateTimeEdit{
border:0px solid #242424;
border-radius:3px;
padding:2px;
color:#FFFFFF;
selection-background-color:#1C1C28;
selection-color:#0EC3FD;
}</string>
       </property>
       <property name="text">
        <string notr="true"/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_2">
     <property name="verticalSpacing">
      <number>6</number>
     </property>
     <item row="0" column="4">
      <widget class="QLabel" name="lab_offset_zero_h2">
       <property name="styleSheet">
        <string notr="true">color:white</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::Panel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="text">
        <string notr="true">0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="TTKCheckButtonWidget" name="cbx_FIR_h2">
         <property name="text">
          <string>FIR</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="TTKCheckButtonWidget" name="cbx_wavelet_h2">
         <property name="text">
          <string>小波</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="2">
      <widget class="QLineEdit" name="lineEdit_H2_NOISE">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>22</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit,QDateTimeEdit{
border:0px solid #242424;
border-radius:3px;
padding:2px;
color:#FFFFFF;
selection-background-color:#1C1C28;
selection-color:#0EC3FD;
}</string>
       </property>
       <property name="text">
        <string notr="true"/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="label_9">
       <property name="styleSheet">
        <string notr="true">color:white</string>
       </property>
       <property name="text">
        <string>参考值</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLineEdit" name="lineEdit_H2_ZERO">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>22</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit,QDateTimeEdit{
border:0px solid #242424;
border-radius:3px;
padding:2px;
color:#FFFFFF;
selection-background-color:#1C1C28;
selection-color:#0EC3FD;
}</string>
       </property>
       <property name="text">
        <string notr="true"/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="lab_zero_h2">
       <property name="text">
        <string>中心偏置</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QLabel" name="lab_offset_noise_h2">
       <property name="styleSheet">
        <string notr="true">color:white</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::Panel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="text">
        <string notr="true">0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="lab_noise_h2">
       <property name="text">
        <string>噪声偏置</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_H2">
       <property name="text">
        <string>TEV2</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="lab_filter_h2">
       <property name="text">
        <string>前置滤波</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TTKCheckButtonWidget</class>
   <extends>QCheckBox</extends>
   <header location="global">Gui/TTK/ttkcheckbuttonwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
